pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "poetry-plugin-export"
-version = "1.1.2"
+version = "1.2.0.dev0"
```

Comment:
I'm not sure if the next version will be 1.2.0 or just 1.1.3. I suppose we can still switch back to 1.1.3 if there are only bugfixes?

Reply:
I think we should go with 1.2.0 given we are bumping the minimum core version in this PR.